src/shared/models/asset.model.ts

```diff
@@ -0,0 +1,47 @@
+import type BigNumber from 'bignumber.js';
+
+import { WalletBlockchains } from './blockchain.model';
+import { Money } from './money.model';
```

Comment:
```suggestion
import type { WalletBlockchains } from './blockchain.model';
import type { Money } from './money.model';
```

where possible, try importing as type